src/OrchardCore.Modules/OrchardCore.Layers/Controllers/AdminController.cs

```diff
@@ -100,8 +100,16 @@ public async Task<IActionResult> Index()
                 {
                     model.Widgets.Add(zone, list = new List<dynamic>());
                 }
+                try
+                {
+                    var widgetDisplay = await _contentItemDisplayManager.BuildDisplayAsync(widget.ContentItem, _updateModelAccessor.ModelUpdater, "SummaryAdmin");
+                    list.Add(widgetDisplay);
+                }
+                catch (Exception _e) {
+                    await _notifier.ErrorAsync(H[_e.Message]);
```

Comment:
We don't translate dynamic messages.
If the exception is specific we can catch it and render a custom message.

In this case logging an error would be fine. Or even better if it's done in the BuildDisplay if it can detect the missing content type. And not throw anything.

Reply:
I will add error but I think that Admin should be notified somehow, shouldn't?. It's clear about dynamic messages, but we have same exception for null content item and null definition. So, a general error string or no notification?